apis/tempo/v1alpha1/tempostack_webhook.go

```diff
@@ -146,6 +146,12 @@ func (d *Defaulter) Default(ctx context.Context, obj runtime.Object) error {
 		r.Spec.Template.QueryFrontend.JaegerQuery.Ingress.Route.Termination = TLSRouteTerminationTypeEdge
 	}
 
+	// if tenant mode is Openshift, ingress type should be route by default.
```

Comment:
@rubenvp8510 should there be a validation for jquery ingeress and gateway ingress? For instance do not allow Jquery ingress if the gateway is deployed?

Reply:
Yes, that is true, we need to validate it.

Reply:
Indeed I think that validation already happens here: https://github.com/grafana/tempo-operator/blob/main/apis/tempo/v1alpha1/tempostack_webhook.go#L254